packages/contracts-bedrock/deploy/000-L2OutputOracle.deploy.ts

```diff
@@ -10,14 +10,8 @@ const deployFn: DeployFunction = async (hre) => {
   const { deployer } = await hre.getNamedAccounts()
   const { deployConfig } = hre
 
-  if (
-    typeof deployConfig.startingTimestamp !== 'number' ||
-    isNaN(deployConfig.startingTimestamp)
-  ) {
-    throw new Error(
-      'Cannot deploy L2OutputOracle without specifying a valid startingTimestamp.'
-    )
-  }
+  const l1 = hre.ethers.getDefaultProvider()
+  const l1StartingBlock = await l1.getBlock(deployConfig.l1StartingBlockTag)
```

Comment:
Its possible that the `l1StartingBlock` is `null`, probably want to handle that case

Reply:
addressed at all usages